scene/resources/label_settings.cpp

```diff
@@ -81,6 +105,23 @@ void LabelSettings::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "shadow_size", PROPERTY_HINT_RANGE, "0,127,1,or_greater,suffix:px"), "set_shadow_size", "get_shadow_size");
 	ADD_PROPERTY(PropertyInfo(Variant::COLOR, "shadow_color"), "set_shadow_color", "get_shadow_color");
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "shadow_offset", PROPERTY_HINT_NONE, "suffix:px"), "set_shadow_offset", "get_shadow_offset");
+
+	ADD_GROUP("Stacked Effects", "");
+	ADD_ARRAY_COUNT("Stacked Outlines", "stacked_outline_count", "set_stacked_outline_count", "get_stacked_outline_count", "stacked_outline_");
+	ADD_ARRAY_COUNT("Stacked Shadows", "stacked_shadow_count", "set_stacked_shadow_count", "get_stacked_shadow_count", "stacked_shadow_");
+
+	stacked_outline_base_property_helper.set_prefix("stacked_outline_");
+	stacked_outline_base_property_helper.set_array_length_getter(&LabelSettings::get_stacked_outline_count);
+	stacked_outline_base_property_helper.register_property(PropertyInfo(Variant::INT, "size", PROPERTY_HINT_NONE, "0,127,1,or_greater,suffix:px"), 0, &LabelSettings::set_stacked_outline_size, &LabelSettings::get_stacked_outline_size);
```

Comment:
Instead of hard-coding defaults, you should create an instance of `StackedOutlineData` and use its properties. See here for example:
https://github.com/godotengine/godot/blob/4e6451d62a931a664acabbf3967e2874fedaadcd/scene/gui/file_dialog.cpp#L1612-L1616